superdesk/publish/subscriber_token.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timedelta
 from superdesk.utc import utcnow
 from superdesk.utils import get_random_token
+from content_api import MONGO_PREFIX
```

Comment:
For example there is great ability to combine wsgi application in one application, but in production depends on requirements it could be run separately:
- http://flask.pocoo.org/docs/0.11/patterns/appdispatch/#combining-applications

for example, with this we could run `content_api` in one WSGI application with superdesk, but it needs more thinking for sure